src/modules/AuthModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,9 +1,278 @@
+
+  const validatePasswordMatch = (value: unknown) => {
+    const password = watch("password");
+    return value === password || "Confirm Password doesn't match Password";
+  };
+
+  function handleMouseDownPassword(event: React.MouseEvent<HTMLButtonElement>) {
+    event.preventDefault();
+  }
+  function handleMouseDownConfirmPassword(
+    event: React.MouseEvent<HTMLButtonElement>
+  ) {
+    event.preventDefault();
+  }
+
+  const backgroundStyle = {
+    backgroundSize: "cover",
+    backgroundPosition: "center",
+    backgroundRepeat: "no-repeat",
+    display: { xs: "none", sm: "flex" },
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    alignItems: "center",
+  };
+
+  const onSubmit: SubmitHandler<FormDataResetPass> = async (data) => {
+    // setLoadingBtn(true);
+    try {
+      const response = await axiosInstance.post(
+        "/admin/users/reset-password",
+        data
+      );
+      toast.success(response.data.message || "Password changed successfully");
+      navigate("/login");
+    } catch (error: any) {
+      toast.error(error.response.data.message || "error");
+    }
+    console.log(data);
+  };
+
   return (
-    <div>ResetPass</div>
-  )
-}
+    <>
+      <ThemeProvider theme={defaultTheme}>
+        <Grid container component="main" sx={{ height: "100vh" }}>
+          <CssBaseline />
+          <Grid
+            item
+            xs={12}
+            sm={6}
+            md={6}
+            component={Paper}
+            elevation={6}
+            square
+          >
+            <h1 className="p-5 text-xl font-serif font-bold">
+              <span className="text-[blue]">Stay</span>cation.
+            </h1>
+            <Box
+              className="px-[0.5em] md:px-[0.5em] lg:px-[5em]"
+              sx={{
+                my: 3,
+                mx: 8,
+                display: "flex",
+                flexDirection: "column",
+                justifyContent: "center",
+              }}
+            >
+              <Typography component="h1" variant="h5">
+                Reset Password
+              </Typography>
+              <p className="mt-4">
+                If you already have an account register <br /> You can{" "}
+                <span className="text-[red]">
+                  <Link to="/login">Login here !</Link>
+                </span>
+              </p>
+              <Box
+                component="form"
+                onSubmit={handleSubmit(onSubmit)}
+                noValidate
+                sx={{ mt: 1 }}
+              >
+                <InputLabel htmlFor="outlined-adornment-email" className="my-1">
+                  email
+                </InputLabel>
+                <OutlinedInput
+                  id="outlined-adornment-email"
+                  fullWidth
+                  type="text"
+                  label="email"
+                  placeholder="Please Type Here"
+                  className="bg-[#F5F6F8] mb-3"
```

Comment:
We need to stick to the colors of the theme. We need to update the place holder as well.